### PR TITLE
fix(DTFS2-5968): Added eligibility criteria versions to e2e tests for portal

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue:
* portal e2e tests were missing eligibility criteria versions which crashed tfm-api

# Changes made: 
* Added versions to all required e2e deal files

### CODE REVIEW

File: e2e-tests/portal/cypress/integration/journeys/maker/submit-a-deal-for-review/dealWithIncompleteAbout.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   },
     "eligibility": {
       "status": "Completed",
+      "version": 4,
```
Comment (IgnatG) [inline e2e-tests/portal/cypress/integration/journeys/maker/submit-a-deal-for-review/dealWithIncompleteAbout.json:20]: shouldn't this be 5?

Comment (Zainzzkk) [reply e2e-tests/portal/cypress/integration/journeys/maker/submit-a-deal-for-review/dealWithIncompleteAbout.json:20]: the other files in this test have version 4, so kept it at 4.  Can change to 5

Comment (IgnatG) [reply e2e-tests/portal/cypress/integration/journeys/maker/submit-a-deal-for-review/dealWithIncompleteAbout.json:20]: v5 please

Comment (Zainzzkk) [reply e2e-tests/portal/cypress/integration/journeys/maker/submit-a-deal-for-review/dealWithIncompleteAbout.json]: done and changed mock data loader to v5

**Review by abhi-markan — APPROVED** [review]
Nice work.
Do we need to do the same for `GEF` deals with `version: 2`?

**Pull request merged**